kibbeh/src/ui/SearchBar.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { SmSolidSearch } from "../icons";
+
+export interface SearchBarProps
+  extends React.ComponentPropsWithoutRef<"input"> {
+  inputClassName?: string;
+}
+
+export const SearchBar: React.FC<SearchBarProps> = ({
+  className,
+  inputClassName,
+  ...props
+}) => {
+  return (
+    <div
```

Comment:
Make the component be a wrapper for the icon and the input, so we can ensure that the icon is `mr-4` apart from the input.

You can set the parent to `bg-gray-700 rounded-lg` (and remove those from the input), remove position absolute from the icon wrapper and set it to `mr-4` and remove `pl-7` from the input and set `bg-transparent`. Let me know if that makes sense, it has to match with https://github.com/benawad/dogehouse/pull/657

Reply:
Pushed some new changes. Can you check again?